lib/table-storage/azure-table.interface.ts

```diff
@@ -38,3 +38,5 @@ export interface Repository<T> {
 
   delete(rowKey: string, entity: T): Promise<AzureTableStorageResponse>;
 }
+
+export type ValueType = ((e: any) => string) | string;
```

Comment:
Just wondering if (e: Function)  is possible, it will help us avoid passing other values.

Reply:
With the current implemetation we can able to validate the return type as string. If we make it as a `Function` then we can't able to validate the return type

Reply:
Ok thanks